userguide/jails.rst

```diff
@@ -430,7 +430,8 @@ of these options.
    |                          |              | `ping(8) <https://www.freebsd.org/cgi/man.cgi?query=ping>`__ and                                                    |
    |                          |              | `traceroute(8) <https://www.freebsd.org/cgi/man.cgi?query=traceroute>`__                                            |
    |                          |              | require raw sockets to operate inside a jail. When set, the source IP addresses are enforced to comply with the     |
```

Comment:
Not at all clear what "the source IP addresses are enforced to comply with the IP address bound to the jail" means.  "Enforced to comply" in particular.